network.go

```diff
@@ -250,7 +255,7 @@ func (net *Network) ConnectHemispheres() {
                 excitatory = true
             }
             toNodes[node] = &ConnInfo{
-                Strength: RandFloat(0.75, 1.75),
+                Strength: RandFloat(0.75, 1.75), // magic numbers
```

Comment:
These could probably be top-of-file constants, which would be useful for people hacking around with this.